doc/source/ray-air/examples/batch_tuning.ipynb

```diff
@@ -0,0 +1,1801 @@
+       "236          DecisionTreeRegressor(max_depth=3)   369.222274   \n",
+       "238          DecisionTreeRegressor(max_depth=3)   384.026867   \n",
+       "207                          LinearRegression()   396.891111   \n",
+       "...                                         ...          ...   \n",
+       "217          DecisionTreeRegressor(max_depth=3)  1326.447774   \n",
+       "5                            LinearRegression()  1417.664071   \n",
+       "6            DecisionTreeRegressor(max_depth=3)  1543.497932   \n",
+       "187                          LinearRegression()  1612.046532   \n",
+       "30                           LinearRegression()  1730.940913   \n",
+       "\n",
+       "                                                    checkpoint  \n",
+       "location_id                                                     \n",
+       "199          Checkpoint(local_path=/home/ray/christy-air/my...  \n",
+       "105          Checkpoint(local_path=/home/ray/christy-air/my...  \n",
+       "236          Checkpoint(local_path=/home/ray/christy-air/my...  \n",
+       "238          Checkpoint(local_path=/home/ray/christy-air/my...  \n",
+       "207          Checkpoint(local_path=/home/ray/christy-air/my...  \n",
+       "...                                                        ...  \n",
+       "217          Checkpoint(local_path=/home/ray/christy-air/my...  \n",
+       "5            Checkpoint(local_path=/home/ray/christy-air/my...  \n",
+       "6            Checkpoint(local_path=/home/ray/christy-air/my...  \n",
+       "187          Checkpoint(local_path=/home/ray/christy-air/my...  \n",
+       "30           Checkpoint(local_path=/home/ray/christy-air/my...  \n",
+       "\n",
+       "[259 rows x 3 columns]"
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Keep only 1 model per location_id with minimum error\n",
+    "final_df = results_df.loc[results_df.groupby(\"location_id\")[\"error\"].idxmin()].copy()\n",
+    "final_df.sort_values(by=[\"error\"], inplace=True)\n",
+    "final_df.set_index(\"location_id\", inplace=True, drop=True)\n",
+    "print(final_df.dtypes)\n",
+    "final_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "24a06818",
+   "metadata": {},
+   "source": [
+    "## Load a model from checkpoint and perform inference  <a class=\"anchor\" id=\"load_checkpoint\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f07b3c31",
+   "metadata": {},
+   "source": [
+    "```{tip}\n",
+    "[Ray AIR Predictors](https://docs.ray.io/en/latest/ray-air/predictors.html) are the intended way of conducting scalable batch inference with Ray AIR.\n",
+    "```\n",
+    "\n",
+    "However, in this notebook, we will just restore a single scikit-learn model directly from checkpoint, and demonstrate it can be used for inference.\n",
```

Comment:
```suggestion
    "In this notebook, we will restore a single scikit-learn model directly from checkpoint, and demonstrate it can be used for inference.\n",
```